src/dags/tree/__init__.py

```diff
@@ -1,9 +1,11 @@
 """Module for handling DAG trees with nested dictionaries and qualified names."""
 
+from __future__ import annotations
+
 from dags.tree.dag_tree import (
     concatenate_functions_tree,
     create_dag_tree,
-    create_input_structure_tree,
+    create_input_types_tree,
```

Comment:
```suggestion
    create_tree_with_input_types,
```
Extra 5 characters are worth it to have reasonable English, I'd say.